ehitchhiking-ui/src/app/pages/main-screen/routes-list/routes-list.component.html

```diff
@@ -27,8 +27,11 @@
         <p class="routes-list-p">
           <span>Driver rating:</span><span>{{ route.driverRating }}</span>
         </p>
+        <p class="routes-list-p">
+          <span>Car:</span><span>{{ route.carsSelect.model }}</span>
+        </p>
         <div class="routes-interface">
-          <mat-checkbox color="primary" [checked]="i < 3 ? true : false" (click)="displayRoute(i)">Display route</mat-checkbox>
+          <mat-checkbox color="primary" [checked]="i < 3 ? true : false" [disabled]="i < 3 ? true : false" (click)="displayRoute(i)">Display route</mat-checkbox>
```

Comment:
magic number 3, use constant

Reply:
i < 3 ? true : false - we should put it into a garbage
i < 3 already returns boolean

Reply:
done